pytorch/build_artifacts/pin_git_modules.py

```diff
@@ -4,6 +4,12 @@
 from os.path import join, dirname
```

Comment:
why is this file moved?

Reply:
talked @arjkesh and we thought it's better to move utility script like this outside of src